admin-site/src/Login.svelte

```diff
@@ -37,22 +40,49 @@
       } else {
         serverURL = window.location.protocol + '//' + window.location.host + '/api/v2';
       }
-      document.getElementById("password").value = "5555";
     }
   }
     
     
    
-  
-  function verifyPassword(){
-    getEventsAndStore();
-    // verifying password logic 
-    var passwordCheck = document.getElementById('password').value;
-    if (passwordCheck == "5555"){
-      pageStateInStore.set(1);
+      
+  async function login(){
+    const payload = { username: usernameFieldValue, password: passwordFieldValue }
+
+    let response = await fetch(serverURL + '/login', {
+      method: 'POST',
+      headers: { 'Content-Type':  'application/json' },
+      body: JSON.stringify(payload) 
+    })
+
+    if(response.status != 200){
+      let errorMessage = await response.text()
+      throw new Error(errorMessage)
     }
   }
 
+  function onLoginButton(event){
+    event.preventDefault()
+
+    login()
+    .then( () => {
+      // happy path!
+      // a cookie named 'jwt' should now be present in this browser
```

Comment:
I'm curious how a cookie is built and handled, I'm assuming that's happening on the server side?